packages/api-admin-users-cognito/src/createAdminUsers.ts

```diff
@@ -94,34 +97,61 @@ export const createAdminUsers = ({
             };
 
             let result;
-            await this.onUserBeforeCreate.publish({ user, inputData: data });
-            /**
-             * Always delete `password` from the user data!
-             */
-            delete (user as any)["password"];
-            try {
-                result = await storageOperations.createUser({ user });
-            } catch (err) {
-                throw WebinyError.from(err, {
-                    message: "Could not create user.",
-                    code: "CREATE_USER_ERROR",
-                    data: { user: result || user }
+
+            const wcpProjectEnvironment = getWcpProjectEnvironment();
+            const postUserSeatsEndpoint = getWcpApiUrl(
+                `/orgs/${wcpProjectEnvironment?.org.id}/projects/${wcpProjectEnvironment?.project.id}/package/seats`
+            );
+            if (wcpProjectEnvironment) {
+                await fetch(postUserSeatsEndpoint, {
+                    method: "POST",
+                    headers: { authorization: wcpProjectEnvironment.apiKey },
+                    body: JSON.stringify({ operation: "increment" })
                 });
```

Comment:
Im pretty sure I need to get the result from this and check the status code, if its failing I need to throw an error here